src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ThemeService .cs

```diff
@@ -1,32 +1,67 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
+using AutoMapper;
+using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api;
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
 using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Security;
+using GovUk.Education.ExploreEducationStatistics.Admin.Validators;
 using GovUk.Education.ExploreEducationStatistics.Admin.ViewModels;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
 using GovUk.Education.ExploreEducationStatistics.Common.Utils;
 using GovUk.Education.ExploreEducationStatistics.Content.Model;
 using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
 using Microsoft.AspNetCore.Mvc;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Validators.ValidationUtils;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.Services
 {
     public class ThemeService : IThemeService
     {
+        private readonly ContentDbContext _context;
+        private readonly IMapper _mapper;
         private readonly IPersistenceHelper<ContentDbContext> _persistenceHelper;
         private readonly IUserService _userService;
         private readonly IThemeRepository _repository;
 
-        public ThemeService(IUserService userService, IThemeRepository repository, 
+        public ThemeService(
+            ContentDbContext context,
+            IMapper mapper,
+            IUserService userService,
+            IThemeRepository repository,
             IPersistenceHelper<ContentDbContext> persistenceHelper)
         {
+            _context = context;
+            _mapper = mapper;
             _userService = userService;
             _repository = repository;
             _persistenceHelper = persistenceHelper;
         }
 
+        public async Task<Either<ActionResult, ThemeViewModel>> CreateThemeAsync(CreateThemeRequest request)
```

Comment:
Perhaps we shouldn't be naming methods 'async' anymore?

Reply:
I agree I will stop using 'async' named methods unless there's a need. I will update the PR.